Gutenberg/version.rb

```diff
@@ -11,8 +11,8 @@
 #
 #   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
 GUTENBERG_CONFIG = {
-  # commit: '',
-  tag: 'v1.97.0-alpha1'
+  commit: '6b3b4f2daf124e06d07256aacf342be6488c66eb'
+  # tag: 'v1.97.0-alpha1'
```

Comment:
A friendly reminder that we will need to update that this to the `v1.97.0` tag once it is published.